APMrover2/Rover.h

```diff
@@ -408,7 +408,7 @@ class Rover : public AP_HAL::HAL::Callbacks {
     // compat.cpp
     void delay(uint32_t ms);
 
-    // control_modes.cpp
+    // mode.cpp
```

Comment:
Not a big deal but normally the list of functions are in alphabetical order by file name.. so I think this should go lower..

Reply:
I understand.